tests/standalone/e2e/ilab_standalone_rhoai_test.go

```diff
@@ -46,14 +61,21 @@ func instructlabDistributedTrainingOnRhoai(t *testing.T, numGpus int) {
 	test := With(t)
 
 	// Pre-requisites :
-
 	rhelaiWorkbenchImage, rhelaiWorkbenchImageExists := GetRhelaiWorkbenchImage()
 	if !rhelaiWorkbenchImageExists {
 		rhelaiWorkbenchImage = ILAB_RHELAI_WORKBENCH_IMAGE
 
 		test.T().Logf("RHELAI workbench image is not provided as environment variable. Using workbench image: %s", ILAB_RHELAI_WORKBENCH_IMAGE)
 	}
 
+	timeout := DEFAULT_TEST_RUN_TIMEOUT
+	inputTimeout, timeoutProvided := GetTimeOut()
+	if timeoutProvided {
+		minutes, err := strconv.Atoi(inputTimeout)
+		test.Expect(err).NotTo(HaveOccurred())
+		timeout = time.Duration(minutes) * time.Minute
```

Comment:
What about using `time.ParseDuration` for greater flexibility? This would allow someone to input values like 2h or 600m, avoiding the limitation of strictly defining everything in minutes.

Reply:
updated, as well as readme, and default